code/client/clrcore-v2/Attributes.cs

```diff
@@ -58,13 +58,13 @@ public CommandAttribute(string command, bool restricted = false)
 		}
 	}
 
-#if !IS_FXSERVER
```

Comment:
Can you add a note above this one describing:
`TODO: revert this commit when RedM has KeyMapping support, also don't make changes to this comment.`

Thanks